static/app/views/issueDetails/groupReplays/useReplaysFromIssue.tsx

```diff
@@ -56,14 +53,12 @@ function useReplayFromIssue({
       name: '',
       version: 2,
       fields: REPLAY_LIST_FIELDS,
-      projects: hasMultiProjectSupport
-        ? [ALL_ACCESS_PROJECTS]
-        : [Number(group.project.id)],
```

Comment:
ya!
this old stuff seems janky. good to go

Reply:
yeah 100p